lib/jsonapi/utils/response/formatters.rb

```diff
@@ -273,7 +273,7 @@ def result_options(records, options)
             end
 
             if JSONAPI.configuration.top_level_meta_include_page_count
-              data[:page_count] = page_count_for(data[:record_count])
+              data[:page_count] = apply_pagination?(options) ? page_count_for(data[:record_count]) : 1
```

Comment:
If `JSONAPI.configuration.top_level_meta_include_page_count` is true but `apply_pagination?` is false, then page count should always equal 1